tsconfig.json

```diff
@@ -10,7 +10,8 @@
         "allowJs": true,
         "resolveJsonModule": false,
         "noUnusedLocals": false,
-        "outDir": "build"
+        "outDir": "build",
+        "skipLibCheck": true
```

Comment:
is this really needed? why?

Reply:
there was a fingerprint generator error thrown.
```
node_modules/browser-pool/dist/browser-pool.d.ts:3:34 - error TS7016: Could not find a declaration file for module 'fingerprint-generator'. '/Users/petr/apify/apify-js/node_modules/fingerprint-generator/src/main.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/fingerprint-generator` if it exists or add a new declaration (.d.ts) file containing `declare module 'fingerprint-generator';`

3 import FingerprintGenerator from 'fingerprint-generator';
                                   ~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error. 
```

Reply:
then just add `@ts-ignore` comment above the import in the pool, that package do not have TS support yet (iirc we already handled this in the PR?)

Reply:
It is there:
```typescript
// @ts-expect-error no types for this package yet
import FingerprintGenerator from 'fingerprint-generator';
```

Reply:
oh i see, it will be removed when compiling, this can be addressed in this script:

https://github.com/apify/apify-js/blob/master/tools/typescript_fixes.js#L208

Reply:
but ideally the library should be rewritten to TS too, as it should probably also go to the SDK monorepo, and i dont want to put there anything else than pure TS projects

Reply:
Oh actually you will need to address this directly in browser pool, that TS fix script is only in SDK.

Would be best to just rewrite it in TS (I can help with that, seems like a job just for few hours), for now you can keep the `skipLibCheck`.

Reply:
I wanted to do that. @Equidem, would you be cool with fingerprint generator in typescript?

Reply:
(if not then at least add a handmade .d.ts file, but we should have it part of the monorepo, hence it should be TS)

Reply:
@petrpatek Of course. But if you need it quickly, somebody else needs to do the rewrite, since it's too much work for me now. Same applies for header generator. Also, if you rewrite it, definitely add me as a reviewer please :)